README.md

```diff
@@ -1,6 +1,138 @@
 # tensorizer
 Module, Model, and Tensor Serialization/Deserialization
 
+## TLDR
+Extremely fast model loads from HTTP/HTTPS and S3 endpoints. GPT-J
+(`20gb`) loads at wire-speed (`~5GB/s`) on a 40gige network, and is
```

Comment:
* `20GB` is preferred instead of `20gb`
* `40GbE` is preferred instead of `40gige`